sklearn/manifold/t_sne.py

```diff
@@ -672,6 +676,12 @@ def __init__(self, n_components=2, perplexity=30.0,
         self.method = method
         self.angle = angle
 
+        # TODO sklearn 0.20: remove this warning
+        if learning_rate == 200.0:
```

Comment:
there does not seem to be a consensus reached, nonetheless I now set it to None hoping this increases the chances the PR gets anywhere.